_sass/layouts/guides.scss

```diff
@@ -277,8 +277,10 @@ div.guides-configuration-reference {
 }
 
 select#guides-version-dropdown {
-  color: black;
-  font-size: 1.7rem;
+  background-color: transparent;
+  color: $white;
+  border: 1px solid $white;
+  font-size: 1.35rem;
```

Comment:
This change doesn't work very well for Guide pages: http://localhost:4000/guides/getting-started-reactive .

Reply:
> This change doesn't work very well for Guide pages: http://localhost:4000/guides/getting-started-reactive .

I've committed more changes do address the two background color situations.